cli/cppcheckexecutor.h

```diff
@@ -99,9 +99,9 @@ class CppCheckExecutor : public ErrorLogger {
     static bool tryLoadLibrary(Library& destination, const std::string& basepath, const char* filename);
 
     /**
-     * Execute a shell command and read the output from it. Returns true if command terminated successfully.
+     * Execute a shell command and read the output from it. Returns exitcode of the command terminated successfully.
```

Comment:
is this sentence by intention?

    Returns exitcode of the command terminated successfully.

Well it needs to be clarified somehow because the exitcode can be nonzero and as I understand it it's always a success exitcode.

Reply:
Just a typo.